site/en/guide/autodiff.ipynb

```diff
@@ -75,7 +75,7 @@
         "[backpropagation](https://en.wikipedia.org/wiki/Backpropagation) for training\n",
         "neural networks.\n",
         "\n",
-        "In this guide, you will explore ways to compute gradients with TensorFlow, especially in [eager execution](eager.ipynb)."
+        "In this guide, you will explore ways to compute gradients with TensorFlow, especially in [eager execution](basics.ipynb)."
```

Comment:
I think this is better if it's not a link at all. The target page doesn't mention eager execution & I can't find a decent alternative, so I think it's better to just leave the text, no link.